src/main/java/icbm/classic/command/IcbmCommand.java

```diff
@@ -0,0 +1,116 @@
+package icbm.classic.command;
+
+import com.mojang.brigadier.CommandDispatcher;
+import com.mojang.brigadier.context.CommandContext;
+import icbm.classic.lib.explosive.ExplosiveHandler;
+import icbm.classic.world.IcbmEntityTypes;
+import lombok.AccessLevel;
+import lombok.RequiredArgsConstructor;
+import net.minecraft.commands.CommandSourceStack;
+import net.minecraft.commands.arguments.ResourceKeyArgument;
+import net.minecraft.core.HolderSet;
+import net.minecraft.core.registries.Registries;
+import net.minecraft.network.chat.Component;
+import net.minecraft.resources.ResourceKey;
+import net.minecraft.world.entity.Entity;
+import net.minecraft.world.entity.EntityType;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.phys.AABB;
+import net.neoforged.neoforge.server.command.EnumArgument;
+
+import java.util.List;
+
+import static com.mojang.brigadier.arguments.DoubleArgumentType.doubleArg;
+import static net.minecraft.commands.Commands.argument;
+import static net.minecraft.commands.Commands.literal;
+
+public class IcbmCommand {
```

Comment:
Avoid lowercasing ICBM, convention would be uppercase in java since ICBM is already shorthand

Reply:
There doesn't seem to be any general code convention on this for Java defined by Oracle. I was just going with what I've used at work and in other projects I have worked on previously.

If we want to stick with uppercase, I can rename those.

Reply:
Oracle no longer defines convention for java and tbh rarely did when they owned it. Given my experience working with their engineers they rarely followed any if they had a convention. Usually I follow industry convention and best practices. Which define that its uppercase for start of each word or fully uppercase for shorthand...  unless inconvenient. Specifically for this project we uppercase ICBM since it is shorthand for Intercontinental ballistic missile.

That said not everything in the project is matching convention and best practices. Its been on the TODO list to get something like PMD running and a common formatter. Along with other tools I commonly implement for clients (context I'm a Sr Engineer/tech lead consultant). As readability convention would be next line for { and space after if... but for now trying to avoid code churn so it is realistic to handle the review process.